examples/converters.py

```diff
@@ -0,0 +1,99 @@
+# This example requires the 'members' privileged intent to use the Member converter.
+
+import typing
+
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.members = True
+
+bot = commands.Bot("!", intents=intents)
+
+
+@bot.command()
+async def userinfo(ctx: commands.Context, user: discord.User):
+    # In the command signature above, you can see that the `user`
+    # parameter is typehinted to `discord.User`. This means that
+    # during command invocation, the `commands` framework will
+    # attempt to convert the value passed for the `user` parameter
+    # to a `discord.User` instance.
```

Comment:
This should mention an example of a value being converted, e.g. IDs, mentions, and names.